android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrColors.kt

```diff
@@ -29,6 +29,26 @@ val textOnSubletrPink
 private val textOnSubletrPinkLight = Color((0xFFFFFFFF))
 private val textOnSubletrPinkDark = Color((0xFFFFFFFF))
 
+val textOnGray
+	@Composable
+	get() = if (isSystemInDarkTheme()) {
+		textOnGrayDark
+	} else {
+		textOnGrayLight
+	}
+private val textOnGrayDark = Color((0xFFFFFFFF))
+private val textOnGrayLight = Color((0xFFFFFFFF))
```

Comment:
With our new dark mode updates, this should now be working. Please make sure that the dark and light mode colors both work on your PR.